tests/tests/swfs/avm2/pixelbender_effect_twirl/twirl.pbk

```diff
@@ -0,0 +1,137 @@
+/*****************************************************************************
+ *
+ * Copyright (C) 2008 Adobe Systems Incorporated
+ * All Rights Reserved.
+ *
+ * NOTICE:  All information contained  herein is,  and remains the property of
+ * Adobe Systems Incorporated and its suppliers, if any.  The intellectual and
+ * technical  concepts  contained  herein  are  proprietary  to  Adobe Systems
+ * Incorporated  and  its suppliers  and may  be covered  by U.S.  and Foreign
+ * Patents, patents in process, and are protected by trade secret or copyright
+ * law.  Dissemination of this information or reproduction of this material is
+ * strictly forbidden  unless prior  written permission is obtained from Adobe
+ * Systems Incorporated.
```

Comment:
😬

Reply:
This seems to be from https://github.com/nojvek/pixelbender/blob/master/twirl/twirl.pbk, which has no such license on it (and it's not like we're distributing it)- where did the license come from?